src/main/java/org/fenixedu/academic/ui/spring/AcademicBaseController.java

```diff
@@ -0,0 +1,69 @@
+package org.fenixedu.academic.ui.spring;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+public class AcademicBaseController {
```

Comment:
A pull request of a feature should not add new conventions, should just stick to the existing ones. Currently error handling is done directly using the spring features, and that is how it should be done until a new convention is taken.